core/src/main/scala/com/databricks/labs/remorph/coverage/estimation/EstimationAnalyzer.scala

```diff
@@ -31,7 +32,15 @@ object SqlComplexity {
 
 case class SourceTextComplexity(lineCount: Int, textLength: Int)
 
-case class EstimationStatistics(allStats: EstimationStatisticsEntry, successStats: EstimationStatisticsEntry)
+case class ParseFailStats(ruleNameCounts: Map[String, Int], tokenNameCounts: Map[String, Int])
```

Comment:
Love this.

Reply:
My 2 cents, we should expose this outside of Estimation Reporter also. 
If some folks directly run the transpile without running estimator. (When we actually build it)

Reply:
Yes - that can be possible. This is the second of a few PRs that are going to improve all of this. So, we will put this here fist, then, having all the mechanisms, it can be propagated to wherever it is useful.